### PR TITLE
chore: dockerize laravel-example

### LINKED ISSUE #2: Add Dockerfile to run this example with a Laravel app
Create a Docker image including a simple Laravel app to test this example package.

### PR DESCRIPTION
**Pull request by poppoerika**
- Added Dockerfile and other files in `docker` directory to run laravel-example app.
- Updated Run the app via Docker section in README to include the instructions.

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,26 @@ Intrigued? Keep on reading!
...
 Build a Docker image for the app:
+```bash
+cd docker
+docker build --tag laravel-example .
+```
+If you run into the API limits from GitHub, add a GitHub personal access token to your composer configuration:
+```bash
+export COMPOSER_AUTH=<YOUR_GITHUB_ACCESS_TOKEN>
+docker build --tag laravel-example --build-arg COMPOSER_AUTH=$COMPOSER_AUTH .
+```
```
Comment (pgautier404) [inline README.md]: Somewhere in here I think we should add a note that the build involves compiling the grpc extension and that it should be expected to take a really long time. I know we were both suspicious of how long it was taking, and I'd like to address that upfront in the instructions.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM php:8.1.12-cli
+ARG COMPOSER_AUTH
+ENV COMPOSER_AUTH=$COMPOSER_AUTH
+
+RUN apt-get update && apt-get install -y -q git rake ruby-ronn zlib1g-dev && apt-get clean
```
Comment (schwern) [inline docker/Dockerfile:5]: What is using rake and ronn in a PHP project?

File: README.md (hunk 1 above)
Comment (poppoerika) [reply README.md]: Good call.

File: docker/Dockerfile (hunk 2 above)
Comment (poppoerika) [reply docker/Dockerfile:5]: It looks like that step is needed to successfully install grpc.

File: README.md [hunk 3]
```diff
@@ -10,9 +10,29 @@ Intrigued? Keep on reading!
...
+- `MOMENTO_CACHE_NAME`=<YOUR_CACHE_NAME>
+- `WEATHER_API_KEY` this is for weather API. Check out [OpenWeather](https://openweathermap.org/) to get an API key.
+
 Build a Docker image for the app:
+```bash
+cd docker
+docker build --tag laravel-example .
+```
+**Notes**: Building the `laravel-example` image involves compiling the PHP gRPC extension.
+And this step to install gRPC is expected to take a long time to complete during build when you first build an image.
```
Comment (pgautier404) [inline README.md]:
```suggestion
**Note**: Building the `laravel-example` image involves compiling the PHP gRPC extension, which will take several minutes to complete.
```

Comment (poppoerika) [reply README.md]: I like your concise version 👍

**Review by pgautier404 — APPROVED** [review]
Looks great, thank you!

**Pull request merged**